doc/UserGuide.md

```diff
@@ -35,10 +35,13 @@ Format: `help`
  
 ### Adding a person: `add`
 Adds a person to the address book<br>
-Format: `add NAME [p]p/PHONE_NUMBER [p]e/EMAIL [p]a/ADDRESS [t/TAG]...` 
+Format: `add NAME [p]p/PHONE_NUMBER [p]e/EMAIL [p]a/BLOCK, STREET, UNIT, PostalCode [t/TAG]...` 
  
 > Words in `UPPER_CASE` are the parameters, items in `SQUARE_BRACKETS` are optional, 
 > items with `...` after them can have multiple instances. Order of parameters are fixed. 
+>
+> The address portion must be entered in a similar fashion with commas and a space behind.
+> All 4 address field are necessary.
```

Comment:
Good effort on updating the documentation. However, you might want to update the examples provided as well.